skyportal/handlers/api/instrument.py

```diff
@@ -208,3 +267,59 @@ def delete(self, instrument_id):
               application/json:
                 schema: Error
         """
+
+
+def add_tiles(instrument_id, instrument_name, regions, field_data):
+    session = Session()
+    try:
+        # Loop over the telescope tiles and create fields for each
+        skyoffset_frames = coordinates.SkyCoord(
+            field_data['RA'], field_data['Dec'], unit=u.deg
+        ).skyoffset_frame()
+
+        ra = np.array([reg.vertices.ra for reg in regions])
+        dec = np.array([reg.vertices.dec for reg in regions])
+        coords = np.stack([ra, dec])
+
+        coords_icrs = coordinates.SkyCoord(
+            *np.tile(coords[:, np.newaxis, ...], (len(field_data['RA']), 1, 1)),
```

Comment:
Describe what the 1,1 is doing here.

Reply:
I added an extra comment here to help explain it is a copy with repeats.